Stanford/JumpstartEngineering/Install/CAPx/CAPxDisplay.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * @file
+ * Define the context for the CAP display view mode for person pages.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\CAPx;
+/**
+ *
+ */
+class CAPxDisplay extends \AbstractInstallTask {
```

Comment:
The modules being used in this case are

context, contextual_view_modes, and capx.  

contextual_view_modes has a nasty configuration array that saves everything that is available in to it. Line 53 is the "meat" of this whole thing.